InteractiveHtmlBom/generate_interactive_bom.py

```diff
@@ -4,6 +4,13 @@
 import argparse
 import os
 import sys
+# Add ../ to the path
+# Works if this script is executed without installing the module
+script_dir = os.path.dirname(os.path.abspath(os.path.realpath(__file__)))
+sys.path.insert(0, os.path.dirname(script_dir))
+# Pretend we are part of a module
+# Avoids: ImportError: attempted relative import with no known parent package
+__package__ = os.path.basename(script_dir)
```

Comment:
Add `__import__(__package__)` after this, needed for py2

Looks good otherwise

Reply:
Ok, done